README.md

```diff
@@ -65,19 +72,19 @@ from mosec.mixin import MsgpackMixin
 logger = get_logger()
 ```
 
-Then, we **build an API** to generate the images for a given prompt. To achieve that, we simply inherit the `MsgpackMixin` and `Worker` class, then override the `forward` method. Note that the input `data` is by default a JSON-decoded object, but `MsgpackMixin` will override it to use [msgpack](https://msgpack.org/index.html) for the request and response data, e.g., wishfully it receives data like `[b'a cut cat playing with a red ball']`. Noted that the returned objects will also be encoded by the `MsgpackMixin`.
+Then, we **build an API** for clients to query a text prompt and obtain an image based on the [stable-diffusion-v1-5 model](https://huggingface.co/runwayml/stable-diffusion-v1-5). To achieve that, we simply inherit the `MsgpackMixin` and `Worker` class, then initialize the model in the `__init__` method and override the `forward` method. Note that the input `data` of `forward` method is by default a JSON-decoded object, but `MsgpackMixin` will override it to use [msgpack](https://msgpack.org/index.html) for both request and response data. In this example, the `forward` method will wishfully receive a _list_ of string, e.g., `['a cute cat playing with a red ball', 'a man sitting in front of a computer', ...]`, aggregated from different clients for _batch inference_, improving the system throughput. Also note that the returned objects will be encoded by the msgpack protocol as well, due to the use of `MsgpackMixin`.
```

Comment:
I think `Note that ...` can be a separate sentence with an explanation about why we need msgpack.